sos/plugins/networking.py

```diff
@@ -71,7 +71,8 @@ def setup(self):
             "/sys/class/net/*/device/numa_node",
             "/sys/class/net/*/flags",
             "/sys/class/net/*/statistics/",
-            "/etc/iproute2"
+            "/etc/iproute2",
+            "/sys/class/net/*/bonding/active_slave"
```

Comment:
Isn't this collected via https://github.com/sosreport/sos/blob/master/sos/plugins/networking.py#L58 that collects `/proc/net/bonding/bond*` ?